app/controllers/minions_controller.rb

```diff
@@ -0,0 +1,15 @@
+# Allows to manage Minions
+class MinionsController < ApplicationController
+  before_action :fetch_minion
+
+  def destroy
+    Orchestration.run kind: :removal, params: { target: @minion.minion_id }
```

Comment:
even though it's prevented from the front-end side, should we make it safer on the api level by adding sth like:

```ruby
def destroy
  Orchestration.run kind: :removal, params: { target: @minion.minion_id }
rescue Orchestration::OrchestrationOngoing
  Rails.logger.debug "Orchestration already ongoing."
ensure
  render nothing: true, status: :no_content
end
```

Reply:
The way that it is it's fine but If we change it, just remember to return a different status instead of no content every time. If an error happens, make it explicit to the client side.

Reply:
Hm... It would be a good idea to explicitly tell the user that another orchestration is going on and not show a generic error message.